Tasks/Src/Task_CheckContactor.c

```diff
@@ -30,11 +32,11 @@ void Task_CheckContactor(void *p_arg) {
     Contactor_On(HVLOW_CONTACTOR);
 
     // Push All Clear message to CAN Q
-    CanMsg.id = ALL_CLEAR;
-    CAN_Queue_Post(CanMsg);
+    CanMsg.id = BPS_ALL_CLEAR;
+    CAN_TransmitQueue_Post(CanMsg);
     // Push Contactor State message to CAN Q
-    CanMsg.id = CONTACTOR_STATE;
-    CAN_Queue_Post(CanMsg);
+    CanMsg.id = BPS_CONTACTOR_STATE;
```

Comment:
previously contactor state was just always a 1, which is why we had this here as a duplicate message (feel free to reorg the code based on this)

Reply:
i.e. you can get rid of this send because we do it periodically now